core/chains/evm/config/toml/defaults/Scroll_Mainnet.toml

```diff
@@ -1,20 +1,22 @@
 ChainID = '534352'
 FinalityDepth = 10
+FinalityTagEnabled = true
 ChainType = 'scroll'
-LogPollInterval = '3s'
+LogPollInterval = '5s'
 MinIncomingConfirmations = 1
 # Scroll only emits blocks when a new tx is received, so this method of liveness detection is not useful
 NoNewHeadsThreshold = '0'
-OCR.ContractConfirmations = 1
 
 [GasEstimator]
-Mode = 'SuggestedPrice'
-# Scroll uses the SuggestedPrice estimator; we don't want to place any limits on the minimum gas price
-PriceMin = '0'
+EIP1559DynamicFees = true
+PriceMin = '1 wei'
+BumpMin = '100 wei'
```

Comment:
based on the current gas Prices the BumpMin is too low, I'd least bump up a couple of gwei

Reply:
happy to modify these values but we'll need to re-run the soak test and get QA signoff. during the tests we didn't encounter any issues with gas but i'm happy to adjust these values

Reply:
Thats is because while running the tests there may not have been gas spikes. when there is a sudden gas spike & if the transaction needs to bump, bumping slowly with 100wei will take quite a long time to get the OCR rounds included. if we are worried about the soak tests, I'd say run a quicker one, since we are only bumping up, validating everything works should be enough